core/FrontController.php

```diff
@@ -107,6 +108,10 @@ private static function generateSafeModeOutputFromError($lastError)
      */
     private static function generateSafeModeOutputFromException($e)
     {
+        StaticContainer::get(LoggerInterface::class)->error('Uncaught exception: {exception}', [
```

Comment:
wouldn't that make the error visible in the UI?

Reply:
Wouldn't it always be visible in the UI? If it's not logged, there won't be information about it in the logs (and it won't trigger fingers crossed).